docs/src/requirements/child-devices.md

```diff
@@ -0,0 +1,157 @@
+# Child Devices
+
+## Child Device Model
+* A child device implies two aspects:
+  - a "physical child-device" on device site
+  - a "cloud child device-twin" (a logical representation of the physical device in the cloud)
+  - ?an edge child device-twin (a logical representation of the physical device in thin-edge)
+* Physical child devices typically are special-purpose hardware devices designed to do just a single task or a handful of tasks at best. 
+* These devices can't (or don't want, out of security reasons) connect to the cloud directly. Instead they get connected to the cloud via a gateway device.
+* The gateway device runs thin-edge, and all physical child devices are connected to the gateway device via network, fieldbus or any other channel. 
+* The gateway device appears in the cloud with it's digital device-twin, with one digital device-twin per each physical child-device underneath.
```

Comment:
The mapping is not always 1:1. There could be multiple physical devices grouped together and represented as a single child device. Similarly, a physical device connected to the cloud may not want a separate device twin in the cloud but just associate itself to the device twin of the gateway itself.

Reply:
Good point. Even a "child device-twin" in the cloud is possible without any "physical child-device" (e.g. when just a local process on the gateway uses the child-device representation).

We might need to put some effort to really explicit define scope and difference of terms "child device-twin", "physical child-device" and the general term "child-device".